setup.py

```diff
@@ -18,7 +18,7 @@ def parse_requirements(filename: str):
 if __name__ == "__main__":
     setup(
         name=PACKAGE_NAME,
-        version="0.5.2",
+        version="0.5.4",
```

Comment:
has to be bumped to .4 because 0.5.3 exists already so release would fail otherwise